From 4edfc1e8a2a7d57b30f9f2f8afba37ffb441aee0 Mon Sep 17 00:00:00 2001
From: YikaiTsai-wiwynn <yikai_tsai@wiwynn.com>
Date: Mon, 9 Sep 2024 13:59:57 +0800
Subject: [PATCH] ARM: dts: aspeed: yosemite4: remove power sensor 0x44 on 11

Change to bind INA238 or ISL28022 in runtime.

lore link: https://lore.kernel.org/all/20240910075357.3531211-1-Delphine_CC_Chiu@wiwynn.com/
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
index 7267ad0657a8..a47e5d2418fb 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
@@ -1087,12 +1087,6 @@ power-sensor@41 {
 		reg = <0x41>;
 	};
 
-	power-sensor@44 {
-		compatible = "ti,ina238";
-		shunt-resistor = <1000>;
-		reg = <0x44>;
-	};
-
 	temperature-sensor@48 {
 		compatible = "ti,tmp75";
 		reg = <0x48>;
-- 
2.34.1

